contracts/deploy/003_liquidity_reward.js

```diff
@@ -27,123 +28,128 @@ const liquidityReward = async ({ getNamedAccounts, deployments }) => {
 
   const assetAddresses = await getAssetAddresses(deployments);
 
-  if (!isMainnet && !isGanacheFork) {
-    // Mock Uniswap pair for OUSD -> USDT is dependent on OUSD being deployed
-    const cOUSDProxy = await ethers.getContract("OUSDProxy");
-
-    const reserve100OUSD = utils.parseUnits("100", 18);
-    const reserve100USDT = utils.parseUnits("100", 6);
-
-    let d = await deploy("MockUniswapPairOUSD_USDT", {
+  await Promise.all(['USDT', 'USDC', 'DAI'].map(async stablecoin => {
```

Comment:
@crazybuster & @tomlinton could you check the code in this file and see it it makes sense to you? I modified the deploy script so it is able to deploy LM contracts for more than 1 stable coin. Since our last plan was to start with 3 stable coins.